### ISSUE TITLE
When expound can't print error (due to conformers), print vanilla spec error

### ISSUE THREAD

**Issue by bhb** [bhb/expound#78]

**Comment by bhb** [comment]
Related: https://github.com/bhb/expound/issues/102#

**Comment by arichiardi** [comment]
I have run into this and thanks for the workaround!

**Comment by bhb** [comment]
@arichiardi  @borkdude I've got a fix for this issue in 0.7.2-SNAPSHOT. If you have time to try it out, please let me know what you think!

**Comment by arichiardi** [comment]
First feedback, with `shadow-cljs` I receive these warnings:

```
------ WARNING #1 --------------------------------------------------------------
 File: expound/problems.cljc:110:47
--------------------------------------------------------------------------------
 107 |                         :clj java.lang.Exception) _e
 108 |                 nil)
 109 |               (catch #?(:cljs :default
 110 |                         :clj java.lang.Error) _e
-----------------------------------------------------^--------------------------
 Use of undeclared Var expound.problems/_e
--------------------------------------------------------------------------------
 111 |                 nil))
 112 | 
 113 |         new-in (cond in1
 114 |                      in1
--------------------------------------------------------------------------------

```

**Comment by borkdude** [comment]
@bhb Is this also available in master?

I've tried http://bit.ly/speculative-repl with 

```
(stest/instrument `subs)
(try
  ;; insert your call here
  (subs "foo" 5)
  (catch :default e
    (println (.-message e))
    (throw e)))
```

and I'm seeing:

> Cannot convert path. This can be caused by using conformers to transform values, which is not supported in Expound

**Comment by bhb** [comment]
This is currently only on the “next” branch. I’m hoping to get a release out this month, which will move this to master.

**Comment by bhb** [comment]
Fixed in 0.7.2-SNAPSHOT (to be released soon)
